src/hooks/useActions.ts

```diff
@@ -0,0 +1,16 @@
+import { bindActionCreators } from "@reduxjs/toolkit";
+import { useMemo } from "react";
+import { useDispatch } from "react-redux";
+import { actions as cartActions } from "../store/cartSlice";
+import { actions as favoriteActions } from "../store/favoritesSlice";
+
+const rootActions = {
+  ...cartActions,
+  ...favoriteActions,
+};
+
+export const useActions = () => {
```

Comment:
I would rather separate actions by store slices. For instance `useCartActions` for `cart`